docs/notebooks/tutorials/04_Omnigenity_Optimization.ipynb

```diff
@@ -0,0 +1,422 @@
+{
```

Comment:
<p>Are these necessary? I think only if you didnt pip install DESC and you're running the notebook from the docs folder</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/PlasmaControl/DESC/pull/535/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='2'/>

Reply:
<p>IDK if they are necessary, but we include this in all of our other notebooks so I copied it over</p>